Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -326,6 +326,45 @@ void IConfigurableHttpClientInitializer.Initialize(ConfigurableHttpClient httpCl
         /// </summary>
         public ICredential UnderlyingCredential => credential;
 
+        /// <summary>Creates a <c>GoogleCredential</c> wrapping a <see cref="ImpersonatedCredential"/>.</summary>
+        /// <param name="TargetPrincipal">The service account to impersonate.</param>
+        /// <param name="Delegates">The chained list of delegates.</param>
+        /// <param name="Scopes">The scopes to request during the authorization grant.</param>
+        /// <param name="LifetimeInSeconds">
```

Comment:
Does this have to be a number of seconds as an integer rather than a `TimeSpan` that we can take the TotalSeconds from? That would be a more conventional way of expressing a duration... although we'd then need to make it a `TimeSpan?` if we wanted the default. (That might be for the best though, so that the default isn't hard-coded into calling code.)

Reply:
Yep made it `TimeSpan?`